test/correctness/simd_op_check.cpp

```diff
@@ -2154,13 +2160,17 @@ class SimdOpCheck : public SimdOpCheckTest {
                 // check("f32x4.demote_f64x2_zero", 4 * w, ???);
 
                 // Single-precision floating point to double-precision
-                check("f64x2.promote_low_f32x4", 2 * w, cast<double>(f32_1));
+                if (Halide::Internal::get_llvm_version() >= 130) {
+                    check("f64x2.promote_low_f32x4", 2 * w, cast<double>(f32_1));
+                }
 
                 // Integer to integer narrowing
-                check("i8x16.narrow_i16x8_s", 16 * w, i8_sat(i16_1));
```

Comment:
If this wasn't failing before, do we need the check?

Reply:
It fails if you build with LLVM12.